src/metabase/server/middleware/offset_paging.clj

```diff
@@ -36,8 +42,20 @@
   [handler]
   (fn [request respond raise]
     (if-let [{:keys [limit offset] :as paging-params} (parse-paging-params request)]
-      (binding [*limit* limit
+      (binding [*limit*  limit
                 *offset* offset
                 *paged?* true]
         (handler (with-paging-params request paging-params) respond raise))
       (handler request respond raise))))
+
+(defn page-result
```

Comment:
Since we're not using `page-result` any more in this implementation, can we leave it out of this PR please?